orcestra_book/plans/HALO-20240907a.md

```diff
@@ -9,11 +9,11 @@ crew:
 - job: HAMP
   name: Clara Bayley
 - job: Dropsondes
-  name: Helene Glöckner
+  name: "Helene Gl\xF6ckner"
 - job: Smart/VELOX
   name: TBD
 - job: SpecMACS
-  name: tbd 
+  name: tbd
```

Comment:
I'd leave this unchanged as well. Generally, I'd recommend not to mix any stylistic changes with actual content changes, as this complicates the reviewing process.

Reply:
good point.. but that wasn't intentional... I am not sure how that appears at times and not at other times

Reply:
It's generally a good idea to add things using `git add -p` (note the `-p`) this is a very quick interactive tool. Using it, you can check all the individual changes prior to committing, and you can easily catch and omit unintended changes.